test/e2e/tests/single_zone_e2e_test.go

```diff
@@ -1011,6 +1011,64 @@ var _ = Describe("GCE PD CSI Driver", func() {
 			Expect(gce.IsGCEError(err, "notFound")).To(BeTrue(), "Expected snapshot to not be found")
 		}()
 	})
+
+	It("Should successfully create RePD from a zonal PD", func() {
```

Comment:
I think we should also add a zonal to zonal disk clone case?

Reply:
Done